Profiles/ProfilesAPI.py

```diff
@@ -1,6 +1,64 @@
-from flask import Blueprint
+from flask import Blueprint, make_response, request
 import sys
 from db import db
+from flask_cors import cross_origin
+
 sys.path.append("../")
 
-profiles_api = Blueprint("profiles", __name__)
\ No newline at end of file
+profiles_api = Blueprint("profiles", __name__)
+
+
+@profiles_api.route("/<int:id>", methods=["GET"])
+@cross_origin(supports_credentials=True)
+def get_profile(id):
+    try:
+        data = {'name': db[id]['name'], 'scores': db[id]['scores']}
```

Comment:
can just 
`data = db[id] `

Reply:
++ if i only POST a new person into db with name only (i.e. {"name" : "test"} ) , this code wont work since that person doesn't have ['scores']